__tests__/management-account-spec.js

```diff
@@ -84,4 +84,48 @@ describe('ManagementAccount', () => {
         })
         .catch(() => {});
     }));
+
+  describe('revokeAll', () =>
+    test('should POST to revoke all tokens of an account', () => {
+      Nylas.accounts.connection.request = jest.fn(() =>
+        Promise.resolve([
+          {
+            success: 'true',
+          },
+        ])
+      );
+      Nylas.accounts
+        .first()
+        .then(account => account.revokeAll())
+        .then(resp => {
+          expect(Nylas.accounts.connection.request).toHaveBeenCalledWith({
+            method: 'POST',
+            path: `/a/${APP_ID}/accounts/revoke_all`,
+          });
+          expect(resp.success).toBe('true');
+        })
+        .catch(() => {});
+    }),
+  
+    test('should POST to revoke all tokens of an account except one token', () => {
+      Nylas.accounts.connection.request = jest.fn(() =>
+        Promise.resolve([
+          {
+            success: 'true',
+          },
+        ])
+      );
+      Nylas.accounts
+        .first()
+        .then(account => account.revokeAll('abc123')) 
+        .then(resp => {
+          expect(Nylas.accounts.connection.request).toHaveBeenCalledWith({
```

Comment:
Please add an assertion that the HTTP request included `keep_access_token=abc123` in the body.